.github/workflows/validate-generated-files.yml

```diff
@@ -1,24 +0,0 @@
-name: validate-generated-files
```

Comment:
Umm, I think this file was deleted/changed by mistake, the changes to the CI action should be done in `release.yml`. Right?